reana_server/config.py

```diff
@@ -38,6 +38,15 @@
 
 REANA_SSO_CERN_CONSUMER_SECRET = os.getenv("CERN_CONSUMER_SECRET", "CHANGE_ME")
 
+REANA_KUBERNETES_DEFAULT_JOBS_MEMORY_LIMIT = os.getenv(
```

Comment:
I was more thinking of creating a new variable instead, and plugging it into `REANA_KUBERNETES_JOBS_MEMORY_LIMIT` as default. Something like this:

```python
REANA_KUBERNETES_DEFAULT_JOBS_MEMORY_LIMIT = '4Gi'
REANA_KUBERNETES_JOBS_MEMORY_LIMIT = os.getenv("REANA_KUBERNETES_JOBS_MEMORY_LIMIT", REANA_KUBERNETES_DEFAULT_JOBS_MEMORY_LIMIT)
```

If it's only going to be used for complexity calculations in reana-server, maybe we can even call it differently, something like `REANA_COMPLEXITY_JOBS_MEMORY_LIMIT` and keep what you have now.